test/index.test.ts

```diff
@@ -15,12 +15,11 @@
  * limitations under the License.
  */
 
-import { promises as fs } from 'fs';
+import plugin = require('../src');
 import postcss from 'postcss';
-import plugin, { Options } from '../src';
 import { toShortName } from '../src/minimal-renamer';
 
-async function run(input: string, options?: Options): Promise<postcss.Result> {
```

Comment:
Losing type-safety here is kind of a bummer :disappointed:.

Reply:
No longer an issue